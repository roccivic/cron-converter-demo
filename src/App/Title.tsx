export const Title = () => (
  <div
    style={{
      padding: 20,
      background: "#333",
      color: "white",
      fontSize: 22,
    }}
  >
    <div className="container">
      <span className="code">cron-converter demo</span>
    </div>
  </div>
);
