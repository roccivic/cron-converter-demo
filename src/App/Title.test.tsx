import { render } from "@testing-library/react";
import { Title } from "./Title";

test("<Title />", () => {
  const comp = render(<Title />);
  expect(comp.container).toMatchSnapshot();
});
