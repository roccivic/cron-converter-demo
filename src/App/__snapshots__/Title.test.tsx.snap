// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> 1`] = `
<div>
  <div
    style="padding: 20px; background: rgb(51, 51, 51); color: white; font-size: 22px;"
  >
    <div
      class="container"
    >
      <span
        class="code"
      >
        cron-converter demo
      </span>
    </div>
  </div>
</div>
`;
